### DISCUSSION TITLE
approximate merit function got worse (-4.507e-03). (convexification is probably wrong to zeroth order)

### DISCUSSION THREAD

**Discussion by dbdxnuliba** [tesseract-robotics/tesseract_ros#213] [General]
![image](https://user-images.githubusercontent.com/13350158/189815164-542b1191-12d6-4f65-aff7-dbef82b4d2a1.png)
![rm_tesseract](https://user-images.githubusercontent.com/13350158/189817077-393d791b-e15f-4983-b804-dc31f81e24d3.gif)
[ERROR] approximate merit function got worse (-4.507e-03). (convexification is probably wrong to zeroth order)

**Comment by Levi-Armstrong** [comment]
That error is normal in most case because the nonlinear system is being approximated by a quadratic. It should be changed to warn.

**Comment by dbdxnuliba** [comment]
ok ,thanks a lot

**Comment by rjoomen** [comment]
Solved by  tesseract-robotics/trajopt#324
